bin/entrypoint

```diff
@@ -4,6 +4,9 @@ set -e
 
 if [ "$1" = "export_statistics_to_json" ]; then
     exec pensieve export-statistics-to-json
+elif [ "$1" = "rerun_config_changed" ]; then
+    exec pensieve rerun_config_changed
 else
     exec pensieve run "$@"
+    exec pensieve rerun_config_changed
```

Comment:
`exec` replaces the shell with the invoked process and never returns, so we can't do it twice.

Apparently using `exec` in Docker entrypoint scripts is important when you're launching a process that needs to handle signals correctly for consistency, but we don't do any special signal handling, so we can just omit `exec`. cf https://stackoverflow.com/a/32261019/892260